std_msgs/Header header
std_msgs/String camera_id
std_msgs/Int8 seq
geometry_msgs/TransformStamped nose
geometry_msgs/TransformStamped torso
geometry_msgs/TransformStamped right_shoulder
geometry_msgs/TransformStamped right_elbow
geometry_msgs/TransformStamped right_hand
geometry_msgs/TransformStamped left_shoulder
geometry_msgs/TransformStamped left_elbow
geometry_msgs/TransformStamped left_hand
geometry_msgs/TransformStamped right_thigh
geometry_msgs/TransformStamped right_knee
geometry_msgs/TransformStamped right_foot
geometry_msgs/TransformStamped left_thigh
geometry_msgs/TransformStamped left_knee
geometry_msgs/TransformStamped left_foot
geometry_msgs/TransformStamped right_eye
geometry_msgs/TransformStamped left_eye
geometry_msgs/TransformStamped right_ear
geometry_msgs/TransformStamped left_ear


geometry_msgs/TwistStamped nose_vel
geometry_msgs/TwistStamped torso_vel
geometry_msgs/TwistStamped right_shoulder_vel
geometry_msgs/TwistStamped right_elbow_vel
geometry_msgs/TwistStamped right_hand_vel
geometry_msgs/TwistStamped left_shoulder_vel
geometry_msgs/TwistStamped left_elbow_vel
geometry_msgs/TwistStamped left_hand_vel
geometry_msgs/TwistStamped right_thigh_vel
geometry_msgs/TwistStamped right_knee_vel
geometry_msgs/TwistStamped right_foot_vel
geometry_msgs/TwistStamped left_thigh_vel
geometry_msgs/TwistStamped left_knee_vel
geometry_msgs/TwistStamped left_foot_vel
geometry_msgs/TwistStamped right_eye_vel
geometry_msgs/TwistStamped left_eye_vel
geometry_msgs/TwistStamped right_ear_vel
geometry_msgs/TwistStamped left_ear_vel

geometry_msgs/AccelStamped nose_acc
geometry_msgs/AccelStamped torso_acc
geometry_msgs/AccelStamped right_shoulder_acc
geometry_msgs/AccelStamped right_elbow_acc
geometry_msgs/AccelStamped right_hand_acc
geometry_msgs/AccelStamped left_shoulder_acc
geometry_msgs/AccelStamped left_elbow_acc
geometry_msgs/AccelStamped left_hand_acc
geometry_msgs/AccelStamped right_thigh_acc
geometry_msgs/AccelStamped right_knee_acc
geometry_msgs/AccelStamped right_foot_acc
geometry_msgs/AccelStamped left_thigh_acc
geometry_msgs/AccelStamped left_knee_acc
geometry_msgs/AccelStamped left_foot_acc
geometry_msgs/AccelStamped right_eye_acc
geometry_msgs/AccelStamped left_eye_acc
geometry_msgs/AccelStamped right_ear_acc
geometry_msgs/AccelStamped left_ear_acc